package.json

```
@@ -7,7 +7,8 @@
     "start": "node bot.js",
     "test": "jest",
     "staging-deploy": "git push heroku-staging master",
-    "prod-deploy": "git push heroku-prod master"
+    "prod-deploy": "git push heroku-prod master",
+		"import-clinics": "source .env && mongoimport -h $MONGODB_URI -d admin -c clinics --type=csv --file=clinics.csv --headerline --columnsHaveTypes"
```

This is super cool! Did not know you could do an import like this directly to the db from the command line! The indentation for this line is off though... make consistent please?
```suggestion
    "import-clinics": "source .env && mongoimport -h $MONGODB_URI -d admin -c clinics --type=csv --file=clinics.csv --headerline --columnsHaveTypes"
```

---

oooooh Github this suggestion feature is COOOL! 🦑